.github/workflows/tests.yml

```diff
@@ -35,6 +34,39 @@ jobs:
           path: tests/fixtures/external-svs
           retention-days: 1
 
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+
+      - name: Install dependencies
+        id: python-dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install flake8==3.8.4
+          python -m pip install black==22.3.0
+          python -m pip install bandit==1.7.0
```

Comment:
could you use pre-commit for this instead?

Reply:
What do you mean? Run precommit in the ci instead of installing the pinned versions of those packages?

Reply:
yes, exactly, or use https://pre-commit.ci/ (its free for open source) :)

Reply:
done, thanks for the suggestion :)